emp/emp-yamls/02_eks-cluster.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: emp.pf9.io/v1alpha1
+kind: EKSCluster
+metadata:
+  name: emp-eks-cluster
+spec:
+  # EKS cluster name as in AWS
+  clusterName: emp-eks-cluster
+  
+  # AWS region where the EKS cluster is running
+  clusterRegion: us-east-1
```

Comment:
Please remove hardcoded values

Reply:
Its a sample template only.. just added for reference..

Reply:
Added a comment with support region list..